crates/bevy_ecs/src/storage/sparse_set.rs

```diff
@@ -31,13 +31,6 @@ impl<I, V> SparseArray<I, V> {
 }
 
 impl<I: SparseSetIndex, V> SparseArray<I, V> {
-    pub fn with_capacity(capacity: usize) -> Self {
```

Comment:
Removed because of dead code presumably?

Reply:
Yep. All of these other removals is because they warned for dead code when made `pub(crate)`.